src/index.d.ts

```diff
@@ -106,6 +106,7 @@ declare module 'minecraft-protocol' {
 		host?: string
 		clientToken?: string
 		accessToken?: string
+		crypto?: Certificates
```

Comment:
I honestly think this should be called certificates instead of crypto to keep it clear what it contains and not as ambiguous to new users.